CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Add a flexible http client providing both contract, and only emulating what's necessary
 - HTTP Client Router: route requests to underlying clients
+- Plugin client and implementations moved here from `php-http/plugins`
```

Comment:
that would do as a wording. we should also update the plugin repo
changelog to point to this move and the new separate repos.